js/iteration.js

```diff
@@ -7,39 +7,42 @@
 export const IterationAPI = superclass =>
     class extends superclass {
         /**
-         * Returns all iterations in the registry.
+         * Returns all iterations for a given project.
          *
          * @memberof IterationAPI
+         * @param {String} projectName The name of the project.
          * @returns {Promise} The iteration result list.
          */
-        async listIterations() {
-            const url = this.baseUrl + "iterations";
+        async listIterations(projectName) {

```

Comment:
move this code into the project naming for this method should be `iterationsProject`